contracts/extensions/StreamingPayments.sol

```diff
@@ -184,6 +191,10 @@ contract StreamingPayments is ColonyExtensionMeta {
       0
     );
 
+    if (getAmountClaimableLifetime(numStreamingPayments) > 0) {
```

Comment:
Is this check necessary? Why would we allow a streaming payment to be created which was worth nothing?

Reply:
We already allowed it, so I had to accommodate it. The alternative would be adding a check instead of this, so it feels like much of a wash to me. In addition, there's nothing stopping you editing a stream you create down to being worth nothing, so preventing creation feels quite arbitrary?